package.json

```diff
@@ -101,6 +104,7 @@
     "@types/core-js": "2.5.5",
     "assert": "2.0.0",
     "async-test-util": "2.0.0",
+    "babel-plugin-replace-import-extension": "^1.1.3",
```

Comment:
Please pin the version, no `^`.
I do not use a package-lock in libs for several reasons, so versions must be pinned.